content/en/docs/ops/best-practices/observability/index.md

```diff
@@ -0,0 +1,268 @@
+the prefix used in the workload-level recording rules (`workload:`). This will allow existing dashboards and
+queries to seamlessly continue working when pointed at the production Prometheus instance (and away from the Istio instance).
+
+Control plane metrics are also collected and federated up to the production Prometheus.
+{{< /tip >}}
+
+### Optimizing metrics collection with recording rules
+
+Beyond just using recording rules to [aggregate over pods and instances](#workload-level-aggregation-via-recording-rules), you may
+want to use recording rules to generate aggregated metrics tailored specifically to your existing dashboards and alerts. Optimizing
+your collection in this manner can result in large savings in resource consumption in your production instance of Prometheus, in
+addition to faster query performance.
+
+For example, imagine a custom monitoring dashboard that used the following Prometheus queries:
+
+  * Total rate of requests averaged over the past minute by destination service name and namespace
+
+    {{< text json >}}
+    sum(irate(istio_requests_total{reporter="destination"}[1m]))
+    by (
+        destination_canonical_service,
+        destination_workload_namespace
+    )
+    {{< /text >}}
+
+  * P95 client latency averaged over the past minute by source and destination service names and namespace
+
+    {{< text json >}}
+    histogram_quantile(0.95,
+      sum(irate(istio_request_duration_milliseconds_bucket{reporter="source"}[1m]))
+      by (
+        destination_canonical_service,
+        destination_workload_namespace,
+        source_canonical_service,
+        source_workload_namespace,
+        le
+      )
+    ) >= 0
+    {{< /text >}}
+
+The following set of recording rules could then be added to the Istio Prometheus configuration, using the `istio` prefix
+to make identifying these metrics for federation simple.
+
+{{< text yaml >}}
+groups:
+- name: "istio.recording-rules"
+  interval: 30s
+  rules:
+  - record: "istio:istio_requests:by_destination_service:rate1m"
+    expr: |
+      sum(irate(istio_requests_total{reporter="destination"}[1m]))
+      by (
+        destination_canonical_service,
+        destination_workload_namespace
+      )
+  - record: "istio:istio_request_duration_milliseconds_bucket:p95:rate1m"
+    expr: |
+      histogram_quantile(0.95,
+        sum(irate(istio_request_duration_milliseconds_bucket{reporter="source"}[1m]))
+        by (
+          destination_canonical_service,
+          destination_workload_namespace,
+          source_canonical_service,
+          source_workload_namespace,
+          le
+        )
+      ) >= 0
+{{< /text>}}
+
+Then configure your production instance of Prometheus to federate from the Istio instance with a
+match clause of `'{__name__=~"istio:(.*)"}'` and a metrics relabeling with `regex: 'workload:(.*)'`.
+
+Finally, update your dashboards to replace the original queries with:
```

Comment:
Technically their dashboard previously would have been the whole "histogram_quantile...." blah blah, so they need to replace the whole thing with simply `avg(istio_request_duration_milliseconds_bucket:p95:rate1m) by (destination_workload)` or similar.

Reply:
Updated to add `avg()`.  Good catch.